doc/devel/coding_guide.rst

```diff
@@ -165,6 +165,48 @@ C/C++ extensions
 
 
 
+PR Review guidelines
+====================
+
+* We run on a 'use your judgement' system (for better or worse) and do
+  not have a strict nose-count rule.  If you have a commit bit, then
```

Comment:
The formal PR review stuff still has a few kinks to work out; outdated comments never disappear, approval is "sticky" even if the submitter changes the PR substantially, etc.

Doesn't waffle auto-label new stuff as "needs review" already? There's definitely "needs review" and "needs revision" columns there now. Or did auto-labelling get disabled along with wafflebot?